### PR TITLE
doc(kepler-problem): #82 add tutorial

### LINKED ISSUE #82: Finish tutorial of Kepler Problem

### PR DESCRIPTION
**Pull request by emptymalei**
Resolves #82

- [x] from_geometry: multiple examples to show case the different orbits
  - [x] circular orbit
  - [x] ellipse 
  - [x] hyperbolic
  - [x] parabolic
- [ ] use iom
- [ ] Maybe some explanations on how the equation was solved.

### CODE REVIEW

File: docs/tutorials/kepler_problem.py [hunk 1]
```diff
@@ -13,22 +13,125 @@
 # ---
 
 # %% [markdown]
-# # Motion in a Central Field
+# # Kepler Orbits
 #
-# In this tutorial, we generate data for objects moving in a central field.
+# In this tutorial, we generate data for kepler problems.
```
Comment (cmp0xff) [inline docs/tutorials/kepler_problem.py:18]:
```suggestion
# In this tutorial we will generate data for the Kepler problem.
```

File: docs/tutorials/kepler_problem.py [hunk 2]
```diff
@@ -13,22 +13,125 @@
...
+    :param dataframe: dataframe containing the trajectory data
+    """
+    fig = px.line_polar(
+        dataframe.assign(
+            phi_degree=dataframe.phi / (2 * np.pi) * 360,
+        ),
+        r="r",
+        theta="phi_degree",
+        hover_data=["t", "r", "phi"],
+    )
```
Comment (cmp0xff) [inline docs/tutorials/kepler_problem.py:81]:
```suggestion
    fig = px.line_polar(
        dataframe.assign(
            phi_degree=dataframe.phi / (2 * np.pi) * 360,
        ),
        r="r",
        theta="phi_degree",
        hover_data=["t", "r", "phi"],
        start_angle=0,
    )
```

**Review by cmp0xff — CHANGES_REQUESTED** [review]
The tutorial shows the trajectory, which is solved by the textbook Kepler problem. However, it does not show our strength in that the trajectory is parameterised by time. For example, can we use colour to indicate the speed?

**Comment by emptymalei** [comment]
> The tutorial shows the trajectory, which is solved by the textbook Kepler problem. However, it does not show our strength in that the trajectory is parameterised by time. For example, can we use colour to indicate the speed?

good point. I added some animations.

**Review by cmp0xff — CHANGES_REQUESTED** [review]
The tutorial shows the basics and our strengths. I can approve it after this round of review.
- We need to fix the floating error when displaying `t`. Currently we see weird numbers like `-3.299999999`, `-3.199999999`, `-3.1`, etc. If I were writing the code, I would divide 1 by a power of 2.
- We can to make the animations more impressive.
  - We need to make them smoother.
  - For periodic orbits, we can do fewer revolutions.
  - For unbounded motion, we should somehow emphasise that the motion near the vertex is faster.

**Review by cmp0xff — APPROVED** [review]

**Pull request merged**